entity-service-client/src/main/java/org/hypertrace/entity/type/service/v2/client/EntityTypeServiceClient.java

```diff
@@ -0,0 +1,43 @@
+package org.hypertrace.entity.type.service.v2.client;
+
+import com.google.common.collect.Lists;
+import io.grpc.Channel;
+import java.util.List;
+import java.util.concurrent.Callable;
+import org.hypertrace.core.grpcutils.client.GrpcClientRequestContextUtil;
+import org.hypertrace.core.grpcutils.client.RequestContextClientCallCredsProviderFactory;
+import org.hypertrace.entity.type.service.v2.EntityType;
+import org.hypertrace.entity.type.service.v2.EntityTypeFilter;
+import org.hypertrace.entity.type.service.v2.EntityTypeServiceV2Grpc;
+import org.hypertrace.entity.type.service.v2.EntityTypeServiceV2Grpc.EntityTypeServiceV2BlockingStub;
+
+public class EntityTypeServiceClient {
```

Comment:
we can probably drop the client - this one doesn't provide any value beyond the grpc generated ones.

Reply:
The way I see, these are convenience methods so that the clients don't have to create requests, etc. I find it pretty useful.

Reply:
It doesn't hurt anything to leave it, but my concern with it is that it pushes a few different decision on the user:
- It makes the context explicit. If I have to pass in tenantId explicitly, then changing the context we propagate through the system becomes more difficult. It also means as a caller I have to unwrap the context, just to pass it to the client to repackage.
- It makes all of the calls using a blocking stub. One of the nice things about grpc is that it generates feature, streaming and blocking stubs right out of the box. Leave it to the caller to decide.
- It prevents using grpc features like deadlines, executors etc.
- It exposes a larger API surface than we've declared in the proto file, by way of convenience methods (e.g. deleteAll - if that should be part of the api, let's declare it in the proto file so we can do breaking change detection against it)

Reply:
:) Done